frontend/packages/dev-console/src/actions/add-resources.tsx

```diff
@@ -158,6 +159,16 @@ export const AddActions: { [name: string]: ActionFactory } = {
     path,
     disabled: accessReviewDisabled,
   }),
+  ProjectAccess: (namespace, application, contextSource, path, accessReviewDisabled) => ({
```

Comment:
Do we need `Sharing` option in the topology context menu? I do not see this AC in the story.